README.md

```diff
@@ -0,0 +1,44 @@
+# Desktop-Version-Portfolio
+- This is a project that involves deploying my portfolio webpage with github pages
+- This project involved making my portfolio responsive and deploying it online for users to access
+- I will be adding media queries to the mobile version portfolio so that it is responsive and adaptable to most screens.
+- These sections of the mobile website include:
+    1. The toolbar section
+    2. Works section
+    3. The About myself
+    4. The contact section
+
+## Built With
+
+- HTML/CSS linters
+- media queries
+
+
```

Comment:
- [ ] Kindy add a demo link to your online version of your portfolio, deployed using GitHub pages. This is a milestone requirement:
![image](https://user-images.githubusercontent.com/61408860/187271730-fb8222df-b22d-4991-8de3-9c280e3fd507.png)